client/my-sites/people/people-list-section-header/index.jsx

```diff
@@ -66,27 +53,10 @@ class PeopleListSectionHeader extends Component {
 			<SectionHeader className={ classes } count={ count } label={ label }>
 				{ children }
 				{ siteLink && (
-					<ButtonGroup>
-						<Button
-							compact
-							href={ siteLink }
-							className="people-list-section-header__add-button"
-							onMouseEnter={ this.showAddTooltip }
-							onMouseLeave={ this.hideAddTooltip }
-							ref="addPeopleButton"
-							aria-label={ translate( 'Invite user', { context: 'button label' } ) }
-						>
-							<Gridicon icon="plus-small" size={ 18 } />
-							<Gridicon icon="user" size={ 18 } />
-							<Tooltip
-								isVisible={ this.state.addPeopleTooltip }
-								context={ this.refs && this.refs.addPeopleButton }
-								position="bottom"
-							>
-								{ translate( 'Invite user', { context: 'button tooltip' } ) }
-							</Tooltip>
-						</Button>
-					</ButtonGroup>
+					<Button compact href={ siteLink } className="people-list-section-header__add-button">
+						<Gridicon icon="user-add" />
+						{ translate( 'Invite People', { context: 'button label' } ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **34** times:
`translate( 'Invite People' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string already exists without a **context**. Only add a context if the meaning of the string is very specific.